### ISSUE TITLE
nvme show-nbft displays nothing

### ISSUE THREAD

**Issue by johnmeneghini** [timberland-sig/nvme-cli#5]
We think the nvme show-nbft should assume the --all parameter is set by default.  The fact that nvme show-nbft shows nothing by default is confusing. 

```
n 3/13/23 2:07 PM, John Meneghini wrote:
> I'm including Lenny as he may know something about this.
>
> Lenny, I am seeing this problem with my POC as well.
>
> Everything works, and I am able to boot with NVMe/TCP, but the nvme show-nbft utility shows now output.
>
> Is there anyway to display the ACPI tables /sys/firmware/acpi/tables/NBFT*

Now that I have John's RH POC running I'm seeing the same thing, i.e. nothing, from show-nbft
even though I know that /sys/firmware/acpi/tables/NBFT exists and has stuff in it.

[root@host-vm ~]# ls -l /sys/firmware/acpi/tables/NBFT
-r--------. 1 root root 724 Mar 15 13:09 /sys/firmware/acpi/tables/NBFT

[root@host-vm ~]# nvme show-nbft
/sys/firmware/acpi/tables/NBFT:
[root@host-vm ~]#

It looks like show-nbft wants options specifying what items you want to
see, e.g.:

[root@host-vm ~]# nvme show-nbft -H -s -d
/sys/firmware/acpi/tables/NBFT:

NBFT Subsystems:

Index Host-NQN                                                                                         Transport Address                                 SvcId HFIs
----- ------------------------------------------------------------------------------------------------ --------- --------------------------------------- ----- --------------------
1     nqn.2014-08.org.nvmexpress:uuid:0c468c4d-a385-47e0-8299-6e95051277db                             tcp       192.168.101.20                          4420  1

NBFT HFIs:

Index Transport PCI Address  MAC Address       DHCP IP Address                              Subnet Mask Bits Gateway                                 DNS
----- --------- ------------ ----------------- ---- --------------------------------------- ---------------- --------------------------------------- ---------------------------------------
1     tcp       0:0:5.0      ea:eb:d3:58:89:58 no   192.168.101.30                          24               0.0.0.0                                 0.0.0.0
[root@host-vm ~]#

IMHO, I think there should be an --all, -a option and that should be the default.

Opinions?

                     -Lenny. 
```

**Comment by mwilck** [comment]
This has been fixed by 23daeed ("nbft: add the nbft show plugin"), which is merged upstream.

@johnmeneghini, please close it.

**Comment by mwilck** [comment]
Closing.
